_posts/2021-10-06-sharedTest.md

```diff
@@ -0,0 +1,134 @@
+---
+title:  "sharedTest pattern: sharing tests and speeding up development"
+author: utzcoz
+---
+
+After [Robolectric's 4.0 released][2], `Robolectric` supports the [`AndroidJUnit4` test runner][3], [`ActivityScenario`][5], and [Espresso][4] for interacting with UI components. As we know, we also can run those tests with official Emulator. This article will show a missed but widely-used pattern called sharedTest to share tests between local and instrumentation tests, and speed up your development with a quicker test response.
```

Comment:
instead of `, and speed up your development with a quicker test response`, I'd say something like `. This will provide the benefit of fast unit testing while ensuring that tests are high-fidelity by enabling them to be run in an emulator`

Reply:
Thanks. Done.